golang/Dockerfile.fedora.slim.tpl

```diff
@@ -0,0 +1,24 @@
+FROM #{FROM}
+
+ENV GO_VERSION #{GO_VERSION}
+
+RUN buildDeps='gcc gcc-c++ git' \
+	&& set -x \
+	&& dnf install -y $buildDeps \
+	&& dnf clean all \
+	&& mkdir -p /usr/local/go \
+	&& curl -SLO "#{BINARY_URL}" \
+	&& echo "#{CHECKSUM}" | sha256sum -c - \
+	&& tar -xzf "go$GO_VERSION.linux-#{TARGET_ARCH}.tar.gz" -C /usr/local/go --strip-components=1 \
+	&& rm go$GO_VERSION.linux-#{TARGET_ARCH}.tar.gz
+
+ENV GOROOT /usr/local/go
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
```

Comment:
Updated to make one layer only.  Chmod is from official Dockerfile, however I think chmod will make sure `go` commands (e.g. go get or go install) will have no issue with permissions to write on $GOPATH